packages/core/src/Cardano/Address/PointerAddress.ts

```diff
@@ -65,7 +68,7 @@ export const CertIndex = (value: number): CertIndex => value as unknown as CertI
  * a stake key by a location on the blockchain of the stake key registration certificate.
  */
 export type Pointer = {
-  slot: Slot;
+  slot: bigint;
```

Comment:
Due to my low experience with pointer addresses I wasn't so sure about this change.
Since @mkazlauskas already approved it, I have no reasons to trust him.